src/sdk/strategy.ts

```diff
@@ -0,0 +1,121 @@
+import { SecretKey } from '@nucypher/nucypher-core';
+import { ethers } from 'ethers';
+
+import { EnactedPolicy } from '../policies/policy';
+import { Alice } from '../characters/alice';
+import { Bob } from '../characters/bob';
+import { Enrico } from '../characters/enrico';
+import { tDecDecrypter } from '../characters/universal-bob';
+
+import { Cohort } from './cohort';
+
+export class Strategy {
+  public policy?: EnactedPolicy;
+  public encrypter?: Enrico;
+  public decrypter?: tDecDecrypter;
+  constructor(
+    public cohort: Cohort,
+    public startDate: Date,
+    public endDate: Date,
+    private alice: Alice,
+    private bob: Bob,
+    private bobSecretKey: SecretKey,
+    public deployed: boolean
+  ) {}
```

Comment:
If we agree on having private constructors, then these fields could be defined in the `constructor`,
```suggestion
  constructor(
    public cohort: Cohort,
    public startDate: Date,
    public endDate: Date,
    private alice: Alice,
    private bob: Bob,
    private bobSecretKey: SecretKey,
    public deployed: boolean,
    public policy?: EnactedPolicy,
    public encrypter?: Enrico,
    public decrypter?: tDecDecrypter,
  ) {}
```